learning_assistant/views.py

```diff
@@ -91,3 +91,41 @@ def post(self, request, course_id):
         status_code, message = get_chat_response(prompt_template, message_list, course_id)
 
         return Response(status=status_code, data=message)
+
+
+class LearningAssistantEnabledView(APIView):
+    """
+    View to retrieve whether the Learning Assistant is enabled for a course.
+
+    This endpoint returns a boolean representing whether the Learning Assistant feature is enabled in a course
+    represented by the course_key, which is provided in the URL.
+
+    Accepts: [GET]
+
+    Path: /learning_assistant/v1/course_id/{course_key}/enabled
+
+    Parameters:
+        * course_key: the ID of the course
+
+    Responses:
+        * 200: OK
+    """
+
+    authentication_classes = (SessionAuthentication, JwtAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, course_id):
+        """
+        Given a course ID, retrieve whether the Learning Assistant is enabled for the corresponding course.
+
+        The response will be in the following format.
+
+            {'enabled': <bool>}
+        """
+        courserun_key = CourseKey.from_string(course_id)
+
+        data = {
+            'enabled': learning_assistant_enabled(courserun_key),
```

Comment:
Note: This will look at the `CourseWaffleFlag` and the `LearningAssistantCourseEnabled` model.